virtualizarr/tests/test_readers/conftest.py

```diff
@@ -0,0 +1,324 @@
+import warnings
+
+import h5py  # type: ignore
+import numpy as np
+import pytest
+import xarray as xr
+from packaging.version import Version
+from xarray.tests.test_dataset import create_test_data
+from xarray.util.print_versions import netcdf_and_hdf5_versions
+
+try:
+    import hdf5plugin  # type: ignore
+except ModuleNotFoundError:
+    hdf5plugin = None  # type: ignore
+    warnings.warn("hdf5plugin is required for HDF reader")
```

Comment:
Why wouldn't this either error or just not be a problem?

Reply:
I added a release note in #307, but I'm still curious about this @sharkinsspatial